dashboard/dashboard-statement-list.md

```diff
@@ -0,0 +1,64 @@
+
+- Visit <http://127.0.0.1:2379/dashboard/#/statement> in your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
+
+All the data shown on the SQL statement summary page are from the TiDB statement summary tables. For more details about the tables, see [TiDB Statement Summary Tables](/statement-summary-tables.md).
+
+### Change Filters
+
+On the top of the SQL statement summary page, you can modify the time range of SQL executions to be displayed. You can also filter the list by database in which SQL statements are executed, or by SQL types. The following image shows all SQL executions over the recent data collection cycle (recent 30 minutes by default).
+
+![Modify filters](/media/dashboard/dashboard-statement-filter-options.png)
+
+### Display More Columns
+
+Click **Columns** on the page and you can choose to see more columns. You can move your mouse to the **(i)** icon at the right side of a column name to view the description of this column:
+
+![Choose columns](/media/dashboard/dashboard-statement-columns-selector.png)
+
+### Sort by Column
+
+By default, the list is sorted by **Total Latency** from high to low. Click on different column headings to modify the sorting basis or switch the sorting order:
+
+![Modify list sorting](/media/dashboard/dashboard-statement-change-order.png)
+
+### Change Settings
+
+On the list page, click the **Settings** button on the top right to change the settings of the SQL statements feature:
+
+![Settings entry](/media/dashboard/dashboard-statement-setting-entry.png)
+
+After clicking the **Settings** button, you can see the following setting page:
+
+![Settings](/media/dashboard/dashboard-statement-settings.png)
+
+On the setting page, you can disable or enable the SQL statements feature. When the SQL statements feature is enabled, you can modify the following settings:
+
+- Collect interval: The length of period for each SQL statement analysis, which is 30 minutes by default. The SQL statements feature summarizes and counts all SQL statements within a period of time. If the period is too long, the granularity of the summary is coarse, which is not good for locating problems; if the period is too short, the granularity of the statistics is fine, which is good for locating problems, but this will result in more records and more memory usage within the same data retention duration. Therefore, you need to adjust this value based on the actual situation, and properly lower this value when locating problems.
+- Data retain duration: The retention duration of summary information, which is 1 day by default. Data retained longer than this duration will be deleted from system tables.
+
+See [Configurations of Statement Summary Tables](/statement-summary-tables.md#configurations) for details.
+
+> **Note:**
+>
+> + Because the statement system table is only stored in memory, after SQL Statements feature is disabled, the data in the system table will be cleared.
```

Comment:
```suggestion
> + Because the statement system table is only stored in memory, after the SQL Statements feature is disabled, the data in the system table will be cleared.
```